tests/Dom/DocumentTest.php

```diff
@@ -400,7 +400,7 @@ public function dataDomDocument()
             ],
             'mustache_url_encoded_attributes_in_template_tags' => [
                 'utf-8',
-                '<!DOCTYPE html><html>' . $head . '<body><template type="amp-mustache"><div><form action="{{action}}"><a href="{{url}}"><img src="{{src}}"></a></form></div></template></body></html>',
+                '<!DOCTYPE html><html>' . $head . '<body><template type="amp-mustache"><div><form action="{{action}}"><a href="{{url}}"><img src="{{    src    }}"></a></form></div></template></body></html>',
                 '<!DOCTYPE html><html>' . $head . '<body><template type="amp-mustache"><div><form action="{{action}}"><a href="{{url}}"><img src="{{src}}"></a></form></div></template></body></html>',
```

Comment:
TBH this confuses me a bit. The srcset is IMHO a configuration of multiple URLs, so encoding the whole configuration looks very odd to me.

The [HTML validator](https://validator.w3.org/nu/#textarea) seems to choke on `%20` inside srcsets.
Pasting
```html
<!DOCTYPE html>
<html lang="en">
<head>
<title>test</title>
</head>
<body>
<img src="/img/needles-1440.jpg" 
       srcset="/img/needles-780.jpg%20780w,
               /img/needles-1024.jpg 1024w"
       sizes="100vw"
       width="780" height="521" 
       alt="testimg">
</body>
</html>
```
gives
![image](https://user-images.githubusercontent.com/1544760/111658592-c883d480-880c-11eb-90d8-8b94352779ef.png)

Reply:
This was meant as an example. It's not actually valid right now. I just used `src` instead of `srcset` since PHP doesn't currently do URL encoding for `srcset`. I was just looking for an example for whitespace in a URL. There's probably a better example, though granted whitespace in URLs is not super common (and perhaps is invalid!).

Reply:
Okay. I could extend the test so that it has a second image that contains spaces, which get replaced. This is hopefully less confusing.

Reply:
I extended the testcase with an image that contains spaces. So I think this is resolved.